### PR TITLE
Upgrade to v0.9.0

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Sample `.pre-commit-config.yaml`:
 
 ```yaml
 -   repo: https://github.com/shellcheck-py/shellcheck-py
-    rev: v0.8.0.4
+    rev: v0.9.0.1
```
Comment (asottile) [inline README.md:30]: this will get updated when I release it -- this should be left alone so I can make a commit for the tag

Comment (asottile) [reply README.md:30]: I'll just do this

Comment (scop) [reply README.md]: Removed in force-pushed 9f8327204d3b517630c9fce9694104d90392c1c3

Comment (asottile) [reply README.md]: oh you got me, race condition :D

**Review by asottile — APPROVED** [review]

**Pull request merged**